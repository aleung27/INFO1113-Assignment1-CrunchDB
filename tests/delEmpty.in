DEL a
SET p 0
GET p
DEL p
DEL p
DEL p
