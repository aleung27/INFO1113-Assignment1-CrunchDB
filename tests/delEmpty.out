> no such key

> ok

> [0]

> ok

> no such key

> no such key

